docs/developer/watcher-local-test.md

```diff
@@ -0,0 +1,136 @@
+# Test watcher locally in two-cluster-setup using K3D
+
+### Contents
+
+### Export customizable attributes
+```shell
+export KLM_PATH=<local-path-of-kyma-lifecycle-manager-project> && \
+export KLM_VERSION=<docker-image-tag-for-local-testing>  && \
+export KUBE_CONFIG_DIR=<path-to-export-local-kubeconfigs-to>
+```
+
+### Create KCP cluster
+Run the following command to create a local control-plane (KCP) cluster:
+```shell
+k3d cluster create kcp-local --port 9080:80@loadbalancer \
+--registry-create k3d-registry.localhost:0.0.0.0:5111 \
+--k3s-arg '--no-deploy=traefik@server:0'
+```
+
+### Install Istio and Manifest CRD
+Run the following command to install Istio and Manifest CRD on the KCP cluster:
+```shell
+kubectl config use-context k3d-kcp-local && \
+istioctl install -y && \
```

Comment:
which Istioctl was this tested with? do we need to specify further here?

Reply:
I think this should be specified

Reply:
the istioctl version or the steps to install istioctl?

Reply:
Maybe demo profile and a link the istioctl installation as reference
`istioctl install --set profile=demo`